Navigators/RootNavigator/RootNavigator.js

```diff
@@ -34,31 +44,66 @@ export const RootNavigator = () => {
     return subscriber; // unsubscribe on unmount
   }, []);
 
+  useEffect(() => {
+    if (error === true) {
+      toast.show({
+        title: 'Login failed',
+        status: 'error',
+        description: error_msg === '' ? 'Please login again' : error_msg,
```

Comment:
Move default error to constants and also error message should be Please try again